app/temp/models/aggregates/aggregateAlertOutfits.model.js

```diff
@@ -6,41 +6,41 @@
  **/
 
 module.exports = (sequelize, Sequelize) => {
-    const AggregateAlertOutfits = sequelize.define("AggregateAlertOutfits", {
+    const AggregateAlertOutfits = sequelize.define('AggregateAlertOutfits', {
         alert_id: {
             type: Sequelize.INTEGER,
             primaryKey: true
         },
         outfit_id: {
-            type: Sequelize.BIGINT.UNSIGNED,
+            type: Sequelize.BIGINT,
```

Comment:
Anything in temp is highly WIP and I've not taken a look at yet, it's coming later on :)

Noted for the future though, this'll be looked at when I've integrated the database layer.

Reply:
@microwavekonijn Some tweaks might need to be made to your data types, Alfa is right, some types need to be numbers.

Reply:
Hmm, I know they are, but I receive them as strings. Not sure if I want to modify them, but I can add a getter.

Reply:
I would say it's pretty much expected that numbers come out as integer / number values from a library, otherwise everyone who uses the library will have to do manipulation on their end.